sources/packages/web/src/views/student/application-offering-change-request/ApplicationOfferingChangeFormView.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+  <full-page-container layout-template="centered-card-tab">
+    <template #header>
+      <header-navigator
+        title="Financial Aid Application"
+        sub-title="View Request"
+        :routeLocation="goBackRouteParams"
+      >
+        <template
+          #buttons
+          v-if="
+            applicationOfferingChangeRequestStatus ===
+            ApplicationOfferingChangeRequestStatus.InProgressWithStudent
+          "
+        >
+          <v-btn
+            v-if="showActionButtons"
+            color="primary"
+            variant="outlined"
+            @click="declineApplicationOfferingChangeRequest"
+            >Decline change</v-btn
+          >
+          <v-btn
+            v-if="showActionButtons"
+            color="primary"
+            class="ml-2"
+            @click="approveApplicationOfferingChangeRequest"
+            >Allow change</v-btn
+          >
+        </template>
+      </header-navigator>
+    </template>
+    <template #tab-header>
+      <v-tabs stacked color="primary"
+        ><v-tab
+          v-for="item in items"
+          :key="item.label"
+          :to="item.command()"
+          :ripple="false"
+        >
+          <span class="mx-2 label-bold"> {{ item.label }} </span>
+        </v-tab>
+      </v-tabs>
+    </template>
+    <router-view />
+    <approve-application-offering-change-request-modal
+      ref="approveApplicationOfferingChangeRequestModal"
+    />
+    <decline-application-offering-change-request-modal
+      ref="declineApplicationOfferingChangeRequestModal"
+    />
+  </full-page-container>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref, computed, onMounted, watchEffect } from "vue";
+import { RouteLocationRaw } from "vue-router";
+import { StudentRoutesConst } from "@/constants/routes/RouteConstants";
+import ApproveApplicationOfferingChangeRequestModal from "@/components/aest/students/modals/ApproveApplicationOfferingChangeRequestModal.vue";
+import DeclineApplicationOfferingChangeRequestModal from "@/components/aest/students/modals/DeclineApplicationOfferingChangeRequestModal.vue";
+import { ModalDialog, useSnackBar } from "@/composables";
+import { ApplicationOfferingChangeRequestService } from "@/services/ApplicationOfferingChangeRequestService";
+import { UpdateApplicationOfferingChangeRequestAPIInDTO } from "@/services/http/dto";
+import { ApplicationOfferingChangeRequestStatus } from "@/types";
+import router from "@/router";
```

Comment:
we use `vue-router`, 
![image](https://github.com/bcgov/SIMS/assets/77353155/72158a9c-2fc0-4571-b645-e52f2b69b237)